src/AutorestSwift/View Models/ServiceClientFileViewModel.swift

```diff
@@ -69,6 +69,7 @@ struct ServiceClientFileViewModel {
         for key in namedOperationGroups.keys {
             namedOperationGroupShortcuts.append(KeyValueViewModel(key: key, value: key.lowercased()))
         }
+        namedOperationGroupShortcuts.sort(by: { $0.key > $1.key })
```

Comment:
Will `namedOperationGroupShortcuts.sorted()` not suffice here?

Reply:
I need to implement KeyValueViewModel to follow 'Comparable' protocol if I switched to sorted.